pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -8015,6 +8018,56 @@
   team_page_full_team_alt_text: "Nearly 100 Code.org employees in matching blue and yellow t-shirts pose in a hotel conference room for the company's summer 2022 Codechella event."
   careers_page_code_pride_alt_text: "Two feminine-presenting employees smile while holding the ends of a Code.org-branded rainbow banner. Behind them, several people are wearing Code.org shirts and waving rainbow flags while walking alongside a multi-colored Volkswagen bug."
 
+  accessibility_hero_heading: "Accessibility at Code.org"
+  accessibility_hero_desc: "We are dedicated to eliminating barriers that prevent the inclusion and full participation of students and educators with disabilities, ensuring that EVERY student has access to computer science."
+  accessibility_efforts_title: "Our Efforts for 2023"
+  accessibility_efforts_desc: "To achieve our goal of a fully accessible platform, we’re implementing a comprehensive action plan that includes:"
+  accessibility_efforts_card_title_1: "Internal accessibility training"
+  accessibility_efforts_card_desc_1: "Our product and engineering teams will complete comprehensive accessibility training by December 2023."
+  accessibility_efforts_card_title_2: "VPAT Analysis"
+  accessibility_efforts_card_desc_2_a: "Code.org will complete an annual"
+  accessibility_efforts_card_desc_2_b: "Voluntary Product Accessibility Template"
+  accessibility_efforts_card_desc_2_c: "to establish a baseline and measure progress annually."
```

Comment:
For strings that have links, be sure to keep it all in one string so things don't get wonky when translated. You can add links inline using this pattern: `[Link copy here](%{link_url})`. 

Here's example of how this works: 
https://github.com/code-dot-org/code-dot-org/blob/8996b0805e8c1b8356736f71cfa33ca0f7db8977/pegasus/sites.v3/hourofcode.com/i18n/en.yml#L707

https://github.com/code-dot-org/code-dot-org/blob/8996b0805e8c1b8356736f71cfa33ca0f7db8977/pegasus/sites.v3/code.org/views/faq_csp.haml#L6